frontend/app/src/components/accounts/manual-balances/ManualBalancesForm.vue

```diff
@@ -107,14 +88,32 @@
       :label="tc('common.location')"
       @blur="v$.location.$touch()"
     />
+
+    <big-dialog
+      :display="showCustomAssetForm"
+      :title="tc('asset_management.add_title')"
+      :action-disabled="!customAssetFormValid || customAssetFormSaving"
+      :primary-action="tc('common.actions.save')"
+      :loading="customAssetFormSaving"
+      @confirm="saveCustomAsset()"
+      @cancel="showCustomAssetForm = false"
+    >
+      <custom-asset-form
+        ref="customAssetForm"
+        :types="customAssetTypes"
+        :edit="false"
+        @valid="customAssetFormValid = $event"
+      />
```

Comment:
This opens a second dialog on top of the first one doesn't it? we can live with it for a bit but we probably need a better approach, I am not really a fan of modals that open from other modals.